sdk/src/main/java/nl/rabobank/gict/payments_savings/omnikassa_frontend/sdk/model/response/OrderStatusResult.java

```diff
@@ -0,0 +1,54 @@
+package nl.rabobank.gict.payments_savings.omnikassa_frontend.sdk.model.response;
+
+import kong.unirest.json.JSONObject;
+
+import nl.rabobank.gict.payments_savings.omnikassa_frontend.sdk.model.Money;
+
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.List;
+import java.util.Spliterator;
+import java.util.stream.Collectors;
+import java.util.stream.StreamSupport;
+
+public class OrderStatusResult {
+    private final String merchantOrderId;
+    private final String id;
+    private final String status;
+    private final Money totalAmount;
+    private final List<TransactionInfoStatus> TransactionInfoOrderStatus;
+
+    public OrderStatusResult(JSONObject jsonObject) {
+        this.merchantOrderId = jsonObject.getString("merchantOrderId");
+        this.id = jsonObject.getString("id");
+        this.status = jsonObject.getString("status");
+        this.totalAmount = Money.fromJsonWithValueInMinorUnits(jsonObject.getJSONObject("totalAmount"));
+        List<TransactionInfoStatus> transactions = new ArrayList<>();
+        if(jsonObject.has("transactions")) {
```

Comment:
There is no test for the false condition of this if statement

Reply:
Done.